ci-operator/config/openshift/release/openshift-release-master__s390x-nightly-4.9.yaml

```diff
@@ -0,0 +1,96 @@
+base_images:
+  ansible:
+    name: "4.9"
+    namespace: ocp
+    tag: ansible
+  assisted-installer:
+    name: assisted
+    namespace: ocp-kni
+    tag: assisted-installer
+  assisted-installer-agent:
+    name: assisted
+    namespace: ocp-kni
+    tag: assisted-installer-agent
+  assisted-installer-controller:
+    name: assisted
+    namespace: ocp-kni
+    tag: assisted-installer-controller
+  assisted-service:
+    name: assisted
+    namespace: ocp-kni
+    tag: assisted-service
+  assisted-test-infra:
+    name: assisted-test
+    namespace: ocp-kni
+    tag: assisted-test-infra
+  aws-ebs-csi-driver-operator-test:
+    name: "4.9"
+    namespace: ocp
+    tag: aws-ebs-csi-driver-operator-test
+  azure-disk-csi-driver-operator-test:
+    name: "4.9"
+    namespace: ocp
+    tag: azure-disk-csi-driver-operator-test
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+  console-tests:
+    name: "4.9"
+    namespace: ocp
+    tag: console-tests
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  gcp-pd-csi-driver-operator-test:
+    name: "4.9"
+    namespace: ocp
+    tag: gcp-pd-csi-driver-operator-test
+  libvirt-installer:
+    name: "4.9"
+    namespace: ocp
+    tag: libvirt-installer
+  upi-installer:
+    name: "4.9"
+    namespace: ocp
+    tag: upi-installer
+  vsphere-csi-driver-operator-test:
+    name: "4.9"
+    namespace: ocp
+    tag: vsphere-csi-driver-operator-test
+releases:
```

Comment:
Define the s390x releases with names that are *not* `latest` and `initial`, then use [dependency overrides](https://docs.ci.openshift.org/docs/architecture/ci-operator/#dependency-overrides) to tell the installer to use the s390x releases for installation

Reply:
I tried updating based on dependency overrides but do you think it also requires anything else to resolve ?